Makefile

```diff
@@ -37,6 +37,10 @@ ifdef VT_EXTRA_BUILD_FLAGS
 export EXTRA_BUILD_FLAGS := $(VT_EXTRA_BUILD_FLAGS)
 endif
 
+ifndef VTROOT
```

Comment:
Why is this change needed?

Reply:
Its used in multiple places in the Makefile, but without it being defined those targets will never work. `make install` and `make install-testing` and `make proto` require it to be defined.